includes/class-wc-payment-token-wcpay-link.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Class WC_Payment_Token_WCPay_SEPA
```

Comment:
```suggestion
 * Class WC_Payment_Token_WCPay_Link
```

Nitpick: comment references SEPA rather than Link

Reply:
done. thank you!